### DISCUSSION TITLE
Default Site Drop connection error

### DISCUSSION THREAD

**Discussion by Seimann** [ZoeyVid/NPMplus#721] [Q&A] [answered]
If I set Default Site to "Drop connection", you can't create new SSL certificates.
(after restarting the docker)

**Comment by Zoey2936** [comment, marked as answer]
![grafik](https://github.com/ZoeyVid/NPMplus/assets/75573284/f4574ad7-6604-4244-89f2-5ab6f6e48bf7)
yes

**Reply by Zoey2936** [reply]
if you drop connections, the ca can not connect to unknown hosts

**Reply by Seimann** [reply]
And cert renew always works?
Thank you for your quick response :)

**Reply by Zoey2936** [reply]
yes renews work
